### DISCUSSION TITLE
canvas.loadFromJSON missing in react-native-canvas

### DISCUSSION THREAD

**Discussion by vtn-dev-manish** [iddan/react-native-canvas#315] [Ideas]
Can we expect a similar method in 'react-native-canvas' as we have in fabric.js.
It also please suggest how we have approach this requirement at our end.

**Comment by swatiredhu17** [comment]
@vtn-dev-manish you found anything related to this? How did you manage it on your end.

**Comment by iddan** [comment]
The library supports the native canvas methods. loadFromJSON is out of scope
